scene/3d/navigation_agent_3d.cpp

```diff
@@ -192,15 +192,15 @@ Vector3 NavigationAgent3D::get_target_location() const {
 Vector3 NavigationAgent3D::get_next_location() {
 	update_navigation();
 	if (navigation_path.size() == 0) {
-		ERR_FAIL_COND_V(agent_parent == nullptr, Vector3());
+		ERR_FAIL_COND_MSG(agent_parent == nullptr,  "The agent has no parent");
```

Comment:
This should be `ERR_FAIL_COND_V_MSG` instead `ERR_FAIL_COND_MSG` - V means that when error happens, something must be returned, it this situation `Vector3()`

Reply:
Updated